backend/config/data_providers.yml

```diff
@@ -1233,7 +1233,7 @@ shared:
     label: API Satelit (Production)
     mailer: *shared_mailer
 
-  moncomptepro:
+  mon_compte_pro:
```

Comment:
MonComptePro c'est un mot unique (comme une marque déposée). Comme franceconnect, agentconnect, etc.

Reply:
Le problème c'est que j'ai nommé ça en `MonComptePro` et quand tu camelize ça fait `mon_compte_pro`.

Du coup au choix:

1. Soit j'itère pour inférer une url différente depuis ce fichier
2. Soit je renomme en `Moncomptepro` les classes dans le code

@rdubigny est-ce que tu as un avis sur la question ?

Reply:
la 2e options me semble idoine.

Reply:
Faisons ça, je modifie et merge dans la foulée du coup. Merci